packages/dmn-js-decision-table/test/spec/expression-language.dmn

```diff
@@ -26,13 +26,13 @@
         <inputEntry id="inputEntry1" expressionLanguage="javascript">
           <text><![CDATA["bronze"]]></text>
         </inputEntry>
-        <inputEntry id="inputEntry2">
+        <inputEntry id="inputEntry2" expressionLanguage="feel">
```

Comment:
Do we still test the default behavior (what if no expression language is set?)

Reply:
I've refactored the tests to make sure we do.